### PR TITLE
Feature - ThongNguyen - Drawing

### PR DESCRIPTION
**Pull request by mvn-thongnguyen-dn**
## DESCRIPTION
Implement excerise of Drawing.
Drawing column chart on subview depend on array of value from superview.

## ScreenShot
IPhone 12 PROMAX
-------

### CODE REVIEW

File: 04_View/ThongNT/View/View/CustomView/Drawing/ColumnChartView.swift [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    }
+    
+    override func draw(_ rect: CGRect) {
+        super.draw(rect)
+        //        drawColumnChart(with: [CGPoint(x: self.bounds.minX , y: self.bounds.maxY),
+        //                               CGPoint(x: 50, y: 50),
+        //                               CGPoint(x: 100, y: 100),
+        //                               CGPoint(x: 150, y: 80),
+        //                               CGPoint(x: 200, y: 180),
+        //                               CGPoint(x: self.bounds.maxX, y: self.bounds.maxY)])
```
Comment (mvn-bienle-dn) [inline 04_View/ThongNT/View/View/CustomView/Drawing/ColumnChartView.swift:33]: Remove unused code

File: 04_View/ThongNT/View/View/CustomView/Drawing/ColumnChartView.swift [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    }
+    
+    private func drawColumnChart(with points: [CGPoint]) {
+        let context = UIGraphicsGetCurrentContext()
+        context?.addLines(between: points)
+        context?.setFillColor(UIColor.systemYellow.cgColor)
+        context?.fillPath()
+    }
+    
+
```
Comment (mvn-bienle-dn) [inline 04_View/ThongNT/View/View/CustomView/Drawing/ColumnChartView.swift:84]: Remove unused line

File: 04_View/ThongNT/View/View/CustomView/Drawing/DrawingViewController.swift [hunk 3]
```diff
@@ -0,0 +1,56 @@
+//
+//  DrawingViewController.swift
+//  View
+//
+//  Created by Thong Nguyen T. VN.Danang on 4/5/22.
+//
+
+import UIKit
+
+struct Values {
```
Comment (mvn-bienle-dn) [inline 04_View/ThongNT/View/View/CustomView/Drawing/DrawingViewController.swift:10]: Rename, need clearly

File: 04_View/ThongNT/View/View/CustomView/Drawing/DrawingViewController.swift [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+        let frame = CGRect(x: view.bounds.minX,
+                           y: view.bounds.midY - tabbarHeight,
+                           width: view.bounds.width,
+                           height: view.bounds.height / 5)
+        columnChartView.datasource = self
+        columnChartView.frame = frame
+        columnChartView.draw(frame)
+        view.addSubview(columnChartView)
+    }
+
```
Comment (mvn-bienle-dn) [inline 04_View/ThongNT/View/View/CustomView/Drawing/DrawingViewController.swift:49]: Remove this line

**Review by mvn-bienle-dn — COMMENTED** [review]
@mvn-thongnguyen-dn Please check my comments.
Thank you!

**Comment by mvn-thongnguyen-dn** [comment]
@mvn-bienle-dn Thank for your comments. 
Please check my fixes.

**Review by mvn-bienle-dn — APPROVED** [review]
LGTM!

**Pull request merged**